plugins/embergen/scripts/installer.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+set -e
+
+# Update packages and install wget
+apt update
+apt install -y wget unzip
+
+# Set local variables
+LAUNCH="$DESTINATION/EmberGen-$VERSION/embergen2"
+ICON="$DESTINATION/EmberGen-$VERSION/embergen.png"
+DESKTOP_FILE="$DESTINATION/EmberGen-$VERSION/embergen.desktop"
+
+# Download and Install the application
+echo "Installing Embergen Version $VERSION into $DESTINATION"
+echo "Downloading Embergen..."
+wget -O "/tmp/embergen-$VERSION-linux.zip" "https://jangafx-software-files.s3.amazonaws.com/embergen/installers/linux/embergen-$VERSION-linux.zip"
+
+echo "Extracting Embergen..."
+unzip -o "/tmp/embergen-$VERSION-linux.zip" -d "$DESTINATION"
+
+chmod -R 555 "$DESTINATION"
+
+# App icon setup
+echo "Adding desktop files"
+cp -v "${PWD}/assets/embergen.png" "$DESTINATION/EmberGen-$VERSION"
+rm -rfv "$DESKTOP_FILE"
+
+echo "[Desktop Entry]
+Version=$VERSION
+Name=Embergen $VERSION
+Comment=Real-time volumetric effects
+Exec=$LAUNCH
+Icon=$ICON
+Terminal=true
+Type=Application
+Categories=X-Polaris
+MimeType=application/x-embergen
+StartupWMClass=Embergen" >> "$DESKTOP_FILE"
+
```

Comment:
Please add another desktop file for GPU. you can reference the nuke plugin for the launch command

Reply:
@renderyourworld 

Ah apologies, this has now changed where we no longer will need to specifically have a GPU desktop file.

https://juno-fx.github.io/Helios/deploy-usage/

We now have the entire Helios container in a VGL wrapper that should resolve the need for us to do this.